commands/FBFlickerCommands.py

```diff
@@ -116,13 +115,14 @@ def inputCallback(self, input):
 
       lldb.debugger.HandleCommand('po [(id)' + oldView + ' ' + recusionName + ']')
     else:
-      print '\nI really have no idea what you meant by \'' + input + '\'... =\\\n'
-
-    viewHelpers.setViewHidden(oldView, False)
+      print '\nChisel - VS Mode: I really have no idea what you meant by \'' + input + '\'... =\\\n'
```

Comment:
Just noticed this. We don't really do this anywhere else with any output. Do you think it's necessary to mention Chisel and VS by name?